server/api/cvrs.py

```diff
@@ -573,7 +591,10 @@ def parse_ballots_file(
         ballots_file: TextIO,
     ) -> Iterator[Tuple[str, CvrBallot]]:  # (CVR number, ballot)
         validate_comma_delimited(ballots_file)
-        ballots_csv = csv.reader(ballots_file, delimiter=",")
+        ballots_file_cleaned = (
+            fix_ess_ballots_file_quoting(line) for line in ballots_file
+        )
+        ballots_csv = csv.reader(ballots_file_cleaned, delimiter=",")
```

Comment:
I do want to be mindful of false positives. Might be helpful to log/record somewhere the updates that were made, if any. Can incorporate that into my next PR